charts/renku/values.yaml

```diff
@@ -60,10 +60,10 @@ ingress:
   ## Annotations for the created ingress
   annotations:
     ## The ingress class
-    kubernetes.io/ingress.class: 'nginx'
+    kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: 'true'
     nginx.ingress.kubernetes.io/proxy-body-size: '0' # Adjust to a reasonable value for production to avoid DOS attacks.
-    nginx.ingress.kubernetes.io/proxy-request-buffering: 'off' # Needed if GitLab is behind this ingress
+    nginx.ingress.kubernetes.io/proxy-request-buffering: off   # Needed if GitLab is behind this ingress
```

Comment:
:/

Reply:
Aaaaaaah!

Reply:
But the problem remains that chartpress will mess up at publish time -_-'

Reply:
but the charts we are using now have been published the same way and deployed without errors...